lib/winapi/Makefile

```diff
@@ -0,0 +1,14 @@
+WINAPI_SRCS := \
+	$(wildcard $(NXDK_DIR)/lib/winapi/*.c)
```

Comment:
I think we should take this opportunity to get rid of the wildcard and list the files explicitly.

Reply:
I did the suggested change.

*I think we could have sorted files when creating user32.lib / kernel32.lib / .. to reduce git diff size. But doing it now is probably good to avoid errors (with unwanted files in winapi.lib), too. So personally it's fine with me eitherway.*